class/Defaults/Notification/Email.php

```diff
@@ -116,4 +116,18 @@ public function send( Triggerable $trigger ) {
 
     }
 
+    /**
+	 * Replace the filtered body with the unfiltered one if the notifications/email/unfiltered_html setting is set to true.
+	 * 
+	 * @param  array $notification_data notification_data from PostData.
+	 * @param  array $ndata ndata from PostData, it contains the unfiltered message body.
+	 * @return array $notification_data with the unfiltered body, if notifications/email/unfiltered_html setting is true.
+     * @filter notification/allow_unfiltered_body
```

Comment:
`@filter ...` should be above the `@param`s. Is the formatting ok or Github brakes it?

Reply:
Sorry, I didn't notice there was a mix of spaces and tab, in my editor I was seeinig this right.